test/import-test.js

```diff
@@ -41,6 +41,29 @@ describe('#importFromFilePath()', () => {
     expectNoConstraints(simple.value);
   });
 
+  it('should correctly import a simple abstract entry', () => {
+    const specifications = importFixture('SimpleAbstract');
+    const simple = expectAndGetEntry(specifications, 'shr.test', 'Simple');
+    expect(simple.grammarVersion).to.eql(new Version(4, 0));
+    expect(simple.concepts).to.have.length(1);
+    expectConcept(simple.concepts[0], 'http://foo.org', 'bar', 'Foobar');
+    expect(simple.description).to.equal('It is a simple entry');
+    expectCardOne(simple.value);
+    expectPrimitiveValue(simple.value, 'string');
+    expectNoConstraints(simple.value);
+  });
+
+  it('should correctly import a simple abstract element', () => {
+    const specifications = importFixture('SimpleAbstractElement');
+    const simple = expectAndGetElement(specifications, 'shr.test', 'Simple');
+    expect(simple.concepts).to.have.length(1);
```

Comment:
Since we're testing abstract elements, I'd suggest you add an `expect` that is looking for the simple's `isAbstract` to be true.  That way if it's not getting set properly, we know!